addons/point_of_sale/models/pos_session.py

```diff
@@ -2167,13 +2171,29 @@ def log_partner_message(self, partner_id, action, message_type):
     def _pos_has_valid_product(self):
         return self.env['product.product'].sudo().search_count(['&', ('available_in_pos', '=', True), ('list_price', '>', 0)], limit=1) > 0
 
+    @api.model
     def _load_onboarding_data(self):
         convert.convert_file(self.env, 'point_of_sale', 'data/point_of_sale_onboarding.xml', None, mode='init', kind='data')
+        shop_config = self.env.ref('point_of_sale.pos_config_main')
```

Comment:
It's possible that the user first opened the pos_restaurant config. In that case, when he chose to load the onboarding data, he won't see the orders. Also, I'm guessing that the main pos config won't actually be open and loading the onboarding data will crash.